Tests/UnitTests/ess-list-encs.component.spec.ts

```diff
@@ -516,6 +516,22 @@ describe('EssListEncsComponent', () => {
     };
     expect(essInfoErrorMessageService.infoErrMessage).toStrictEqual(warnObj);
   });
+
+  test('getEstimatedTotalSize should return esimatedTotalSize when selectedEncList length is 0', () => {
```

Comment:
Please correct the spelling mistake esimatedTotalSize to estimatedTotalSize

Reply:
Resolved